pkg/steps/pod_test.go

```diff
@@ -351,3 +394,36 @@ func expectedPodStepTemplate() *podStep {
 		},
 	}
 }
+
+func TestGetEnv(t *testing.T) {
```

Comment:
We don't need this func so we can drop the test

Reply:
Done.